docs/sphinx/precompiling.md

```diff
@@ -0,0 +1,95 @@
+# Precompiling
+
+Precompiling is compiling Python source files (`.py` files) into byte code (`.pyc`
+files) at build
+time instead of runtime. Doing it at build time can improve performance by
+skipping that work at runtime.
+
+Precompiling is enabled by default, so there typically isn't anything special
+you must do to use it.
```

Comment:
PR message and changelog say it's _disabled_ by default?

Reply:
Oops, thanks for catching that. I've sent a PR to fix the docs here. It is disabled by default for now. After the next release, we can consider enabling it by default